### ISSUE TITLE
product info page: Add to Cart form name and id mismatch

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#5637]
```
<form 
name="cart_quantity" 
action=".................action=add_product" 
method="post" 
enctype="multipart/form-data" 
id="addToCartForm">
```

It gets flagged by the validator. 
"cart_quantity" is not really a logical name for the form and could be the same as the id.

Additionally I note that inside the form, the text field for the Add to Cart quantity is also named "cart_quantity".
While names are not required to be unique, it seems more logical to keep this and change the form name.

**Comment by proseLA** [comment]
which validator and which template?  

using responsive classic on that page, i get no validator elements flagged similar to what you are saying:

![image](https://user-images.githubusercontent.com/1095136/224346402-25b9ac82-bf47-49a2-8d50-287ed69dc5fa.png)

**Comment by torvista** [comment]
html validator in the Firefox developer tools
![image](https://user-images.githubusercontent.com/4391026/224359128-f54b674e-858c-4dfd-9607-d4f1f52c9dc0.png)

**Comment by torvista** [comment]
> changing the form's name could have unwanted repercussions on product-page plugins that monitor for that specific form-name.
Sure, like DPU does. But easily changed with an admin switch. Plugins have to adapt to the core, not the other way round.

I was pointing it out only.

**Comment by proseLA** [comment]
and i would like to point out that you are using an extension, only available for firefox, that does NOT look to validate html 5 in offline mode.  on the doc page, it specifically says: 

[- W3 Online validator to validate HTML 5 pages](https://tinyurl.com/2qfb97ka)

on that same page, it also states:

- The SGML parser is the same program than the one running behind [validator.w3.org](https://prod.outgoing.prod.webservices.mozgcp.net/v1/84bd875adc3539006d171c969c5a930a1f8bc0db8a1f4b48a01b6f34c5d0737d/http%3A//validator.w3.org).

and yet, when i really examine the page, i see 3 different things when validating a product page.

[screencast of html validation using offline, w3c online, and then copying source and validating online at validator.w3.org.](https://www.dropbox.com/s/t7vtfn664qkpjwf/html_validator.webm?dl=0)

while i do not doubt it's usefulness, i would suggest vetting such validation errors using the online validator at https://validator.w3.org/.

**Comment by scottcwilson** [comment]
Yes - I would only consider the w3 validator to be authoritative.

**Comment by torvista** [comment]
I agree with both the above.
Still, while not invalid code, I still think the points made in my first post are valid.
